### ISSUE TITLE
<html> in GitHub Star button violates WCAG 3.1.1

### ISSUE THREAD

**Issue by auchenberg** [mdo/github-buttons#130]
Hi,

We are using these buttons on the VS Code website, and per our accessibility testing we have found:

html-has-lang: <html> element must have a lang attribute (1) WCAG 3.1.1

Url: https://code.visualstudio.com/docs/editor/versioncontrol

Links:
- https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html
- https://dequeuniversity.com/rules/axe/3.2/html-has-lang?application=msftAI

Code path:
```
<html>
```

Downstream issue, https://github.com/microsoft/vscode-docs/issues/2617
